packages/FlexGrid/Row/Row.jsx

```diff
@@ -51,13 +73,38 @@ Row.propTypes = {
    * Distribute empty space around columns.
    */
   distribute: PropTypes.oneOf(['around', 'between']),
+  /**
+   * Choose if the item order should be reversed within the 'xs' breakpoint range.
```

Comment:
`true` and `false` values have different effects here. It may be worth explaining them more:

>Choose whether item order should be reversed _from_ the `xs` breakpoint.